kolibri_instant_schools_plugin/assets/src/views/sign-in-page/index.vue

```diff
@@ -2,8 +2,9 @@
 
   <div class="wrapper-table">
     <div class="main-row"><div id="main-cell">
-      <logo class="logo"/>
-      <h1 class="login-text title">{{ $tr('kolibri') }}</h1>
+      <!-- swap core logo? -->
+      <img class="logo" src="../img/instant-schools-logo.png">
```

Comment:
Using `<img>`, but I've been noticing a trend towards setting div backgrounds. Any reason for that? Should I swap?

Reply:
> Using `<img>`

sounds right

> I've been noticing a trend towards setting div backgrounds

I can't find any examples of this - can you link?

Reply:
sure! here's how we're defining the background in the vf plugin: 
https://github.com/fle-internal/kolibri-instant-schools-plugin/pull/63/files#diff-2f1a826265834d888ff6f79cc8413aabR30

This is how we're doing the loading spinner in kolibri: https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/views/loading-spinner/index.vue#L54

and here's a PR to make it the standard way to add an icon to `app-bar`:
https://github.com/learningequality/kolibri/pull/2487/files#diff-da5b10cbd6e892779dde720a4cf88b43R173